api/src/main/resources/templates/pdfviewer/pdfviewer.vm

```diff
@@ -131,9 +131,9 @@
   #end
 #end
 
-#set ($mainPageReference = $services.model.createDocumentReference('', ['XWiki'], 'PDFViewerService'))
+#set ($mainPageReference = $services.model.createDocumentReference('', ['PDFViewer', 'Code'], 'WebHome'))
```

Comment:
I see the same was already done for the other licensure check in [this commit](https://github.com/xwikisas/macro-pdfviewer/commit/6774a448808a5523f704ebb303628337a89ccaff). So now there is consistency, even if there was no problem with the `PDFViewerService` file (it's not a license excluded document)
But you should add a comment on the PR about why this change what needed

Reply:
This change is not related to the error.
I modified it to keep consistency throughout the code, because at [https://github.com/xwikisas/macro-pdfviewer/blob/master/wiki/src/main/resources/PDFViewer/WebHome.xml](url) it is used the updated version.